governance/xc_admin/packages/crank_pythnet_relayer/src/index.ts

```diff
@@ -101,16 +97,19 @@ async function run() {
       ) {
         const preInstructions: TransactionInstruction[] = [];
 
-        console.log(`Found VAA ${lastSequenceNumber}, relaying ...`);
+        console.log(`Found VAA ${lastSequenceNumber}, relaying vaa ...`);
+
         await postVaaSolana(
           provider.connection,
           signTransactionFactory(KEYPAIR),
           WORMHOLE_ADDRESS[CLUSTER]!,
           provider.wallet.publicKey,
           Buffer.from(response.vaaBytes, "base64"),
-          { commitment: COMMITMENT }
+          { commitment: COMMITMENT, maxRetries: 10 }
```

Comment:
I couldn't see what the default is. It was often failing at this step.

Reply:
meeeh, people in Solana recommend 0 retries at this level
`https://github.com/rpcpool/optimized-txs-examples/blob/main/selfSolTransfer.mjs#L116`
instead if you'd like to retry you should put `postVaaSolana` inside